crates/hyperswitch_connectors/src/connectors/santander.rs

```diff
@@ -830,27 +948,59 @@ impl ConnectorIntegration<RSync, RefundsData, RefundsResponseData> for Santander
     }
 }
 
+fn get_webhook_object_from_body(
+    body: &[u8],
+) -> CustomResult<santander::SantanderWebhookBody, common_utils::errors::ParsingError> {
+    let webhook: santander::SantanderWebhookBody = body.parse_struct("SantanderIncomingWebhook")?;
+
+    Ok(webhook)
+}
+
 #[async_trait::async_trait]
 impl webhooks::IncomingWebhook for Santander {
-    fn get_webhook_object_reference_id(
+    async fn verify_webhook_source(
         &self,
         _request: &webhooks::IncomingWebhookRequestDetails<'_>,
+        _merchant_id: &common_utils::id_type::MerchantId,
+        _connector_webhook_details: Option<common_utils::pii::SecretSerdeValue>,
+        _connector_account_details: crypto::Encryptable<Secret<serde_json::Value>>,
+        _connector_name: &str,
+    ) -> CustomResult<bool, errors::ConnectorError> {
+        Ok(true)
```

Comment:
Please add a comment here for hardcoding `Ok(true)` here mentioning the current integration limitations